### PR TITLE
#183 Relationship Matrix selected Categories display.

### LINKED ISSUE #183: External display of categories in the Relationship Matrix does not follow alphabetic order.
### Prerequisites

- [ X] I have written a descriptive issue title
- [X ] I have verified that I am running the latest version of the CDP4-IME
- [ X] I have searched [open](https://github.com/RHEAGROUP/CDP4-IME-Community-Edition/issues) and [closed](https://github.com/RHEAGROUP/CDP4-IME-Community-Edition/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description
The external display of categories in the Relationship Matrix does not follow an alphabetic order, which is followed inside the category selector checkbox.

### Steps to Reproduce
Select multiple same categories for both X and Y Axis, see the category that is displayed from the outside of the combo box does not correspond with the one that alphabetically should be. In the file attached, the same categories have been selected for both axis, but in the X-Axis the one displayed from the outside is "Mission Analysis Descriptor", and in the Y-Axis "Communications Descriptor", but if following the order they appear in the checkbox, the first one for both X and Y-Axis should be "AOGNC Descriptor".

This can lead to a misunderstanding, in the example, that in the X-Axis "Communications Descriptor" has not been selected since the one that appears from the outside is Mission Analysis, which would come later in the alphabet.

### Logs

  - [X ] There are no logs regarding the issue
  - { ] I have attached a log file or included the relevant section in this issue  
  
### System Configuration

- CDP4 version:
  - [X
Capture
 ] CDP4-IME: 
  - [ ] Other: 
  -
- Web Services:
  - [ ] CDP4 Web Services version: 
  - [ ] OCDT WSP version: 
- Environment (Operating system, version and so on):
  - [ ] Microsoft Excel (32/64 bit): 
  - [ ] Windows OS (32/64 bit): 
  - [ ] Additional information:

### PR DESCRIPTION
**Pull request by alexatrhea**
### Prerequisites

- [ ] I have written a descriptive pull-request title
- [ ] I have verified that there are no overlapping [pull-requests](https://github.com/RHEAGROUP/CDP4-IME-Community-Edition/pulls) open
- [ ] I have verified that I am following the CDP4-IME [code style guidelines](https://raw.githubusercontent.com/RHEAGROUP/CDP4-IME-Community-Edition/master/.github/CONTRIBUTING.md)
- [ ] I have provided test coverage for my change (where applicable)

### Description
Selected Categories in the Relationship Matrix now displayed in alphabetical order instead of by order of selection. - fixes #183

### CODE REVIEW

File: RelationshipMatrix/Converters/OrderedCategoryListConverter.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        /// </param>
+        /// <param name="culture">
+        /// The culture information.
+        /// </param>
+        /// <returns>
+        /// The <see cref="object"/> containing the same objects as the input collection.
+        /// </returns>
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            return ((IList) value)?.Cast<Category>().OrderBy(c => c.Name).ToList() ?? new List<Category>();
```
Comment (lxatrhea) [inline RelationshipMatrix/Converters/OrderedCategoryListConverter.cs]: Maybe begin every next function on a new line. 
Makes it a bit more readable.

File: RelationshipMatrix/Converters/OrderedCategoryListConverter.cs [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        /// </param>
+        /// <param name="culture">
+        /// The culture information.
+        /// </param>
+        /// <returns>
+        /// The <see cref="List{T}"/> of <see cref="Category"/> containing the same objects as the input collection, but ordered.
+        /// </returns>
+        public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            return ((IList)value)?.Cast<Category>().OrderBy(c => c.Name).ToList() ?? new List<Category>();
```
Comment (lxatrhea) [inline RelationshipMatrix/Converters/OrderedCategoryListConverter.cs]: Maybe begin every next function on a new line. 
Makes it a bit more readable.

Comment (alexatrhea) [reply RelationshipMatrix/Converters/OrderedCategoryListConverter.cs]: When a line does not exceed certain length I actually prefer one-line, reads like a sentence for me.

File: RelationshipMatrix/Converters/OrderedCategoryListConverter.cs (hunk 1 above)
Comment (alexatrhea) [reply RelationshipMatrix/Converters/OrderedCategoryListConverter.cs]: When a line does not exceed certain length I actually prefer one-line, reads like a sentence for me.

**Pull request merged**